CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+# dbt_twitter_source v0.8.0
+
+[PR #27](https://github.com/fivetran/dbt_twitter_source/pull/27) includes the following **BREAKING CHANGE** updates:
+
+## Feature Updates: Native Conversion Support
+We have added more robust support for conversions in our data models by doing the following:
+- Created the `twitter_ads__conversion_fields` and `twitter_ads__conversion_sale_amount_fields` variables to pass through conversion metrics (total number and monetary value, respectively).
```

Comment:
I recommend to add a short explanation why they are separated and can't be a part of the existing passthrough variable. Is it just since they need to be for the transform package?